code/modules/mob/living/carbon/shock.dm

```diff
@@ -8,8 +8,7 @@
 	1	* src.getToxLoss() + 		\
 	1	* src.getFireLoss() + 		\
 	1	* src.getBruteLoss() + 		\
-	1	* src.getCloneLoss() + 		\
-	1	* src.halloss
+	1	* src.getCloneLoss()
```

Comment:
@PsiOmegaDelta 

I totally forgot that I did that, I's a dumb fox.

Though, it still doesn't explain, this, evne in yours:

`1   * src.getOxyLoss() +        \`